sdk/core/azure-core/CHANGELOG.md

```diff
@@ -10,6 +10,9 @@
 
 ### Other Changes
 
+- Removed `_internal` APIs from the public API surface. Also removed most of the `_internal` APIs from the public `tracing.hpp` header.
+
```

Comment:
If we're releasing this as GA, let's repeat the Beta 1 message for this release changelog instead of listing breaking changes.
Stable releases do not normally have breaking changes, and this I think will create an unnecessary alert for to customer, especially that telemetry is mostly internal, few are using it, and most were using the GA anyway.
```suggestion
- Added prototypes and initial service support for Distributed Tracing.

```